sql/src/main/java/io/crate/analyze/relations/QueriedRelation.java

```diff
@@ -26,4 +26,9 @@
 public interface QueriedRelation extends AnalyzedRelation {
 
     QuerySpec querySpec();
+
+    // relation Id used to distinguish between relations in the Fetch phase (union)
+    byte relationId();
+
+    void relationId(byte relationId);
```

Comment:
We assign relation ids and we increment even for queriedselectrelations that "disappear" after rewriting in RelationNormalizer